types/sse.d.ts

```diff
@@ -1,84 +1,82 @@
 export type SSE = {
-  /** Constructor. */
-  new (url: string, options?: SSEOptions): SSE;
```

Comment:
I believe preserving this constructor definition is important, even though it is (somehow) not getting automatically generated.